content/millennium/r4/individuals/related-person.md

```diff
@@ -20,6 +20,8 @@ The following fields are returned if valued:
 * [Active](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.active){:target="_blank"}
 * [Patient](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.patient){:target="_blank"}
 * [Relationship](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.relationship){:target="_blank"}
+* [Relation](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.relationship){:target="_blank"}
+* [Period](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.relationship){:target="_blank"}
```

Comment:
Should these just be listed with the other extensions below? Or they could be in there own bullet with `Relationship extensions including Related and Period`

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/624/commits/95b78bfc03c61933a3a7b57577a122f4bf27dae5#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR29
https://github.com/cerner/fhir.cerner.com/pull/624/commits/95b78bfc03c61933a3a7b57577a122f4bf27dae5#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR39

Reply:
Need a comma after 'relationship level'.

Reply:
added here https://github.com/cerner/fhir.cerner.com/pull/624/commits/84711bc6d51468385717ab121588f145e0a9bf3b